subprojects/language-native/src/main/java/org/gradle/language/swift/plugins/SwiftBasePlugin.java

```diff
@@ -137,6 +142,40 @@ public String call() {
                     link.setTargetPlatform(currentPlatform);
                     link.setToolChain(toolChain);
                     link.setDebuggable(binary.isDebuggable());
+                } else if (binary instanceof SwiftBundle) {
+                    // Add a link task
+                    LinkBundle link = tasks.create(names.getTaskName("link"), LinkBundle.class);
+                    link.source(compile.getObjectFileDirectory().getAsFileTree().matching(new PatternSet().include("**/*.obj", "**/*.o")));
+                    link.lib(binary.getLinkLibraries());
+                    link.setLinkerArgs(Collections.<String>emptyList());
```

Comment:
this is the default now for the task, and isn't required.

Reply:
Thanks, removed.